pkg/apis/operator/v1alpha1/validation/garden.go

```diff
@@ -48,7 +48,12 @@ import (
 )
 
 var (
-	gardenCoreScheme *runtime.Scheme
+	gardenCoreScheme                      *runtime.Scheme
+	forbiddenShootOperationsToRunTogether = map[string]sets.Set[string]{
```

Comment:
```suggestion
	forbiddenShootOperationsToRunInParallel = map[string]sets.Set[string]{
```

same naming scheme like in `validation/shoot.go`?

Reply:
I have opted to not name this map `forbiddenShootOperationsToRunInParallel` in both `shoot` and `garden` validation, since it can be confusing given that these operations can run in parallel but cannot run together in parallel.

I do not have a strong opinion on this naming and can change it.

Reply:
What about something like `incompatibleShootOperations` or `incompatbileShootOperationsToRunInParallel`

Reply:
With the latest commit I have renamed it to `incompatibleShootOperations` and `incompatibleOperationAnnotations` for the `Garden` validation.